ml/src/main/scala/ai/h2o/sparkling/ml/algos/H2OAutoML.scala

```diff
@@ -90,6 +90,8 @@ class H2OAutoML(override val uid: String)
 
     val algoName = getLeaderboard().select("model_id").head().getString(0)
     deleteRegisteredH2OFrames()
+    train.delete()
```

Comment:
should you instead call registerFrameForDeletion? and let the frames be cleaned up by deleteRegisteredH2OFrames?

Reply:
Like this idea!